components/cache/readme.md

```diff
@@ -0,0 +1,21 @@
+# <img src="../../torch-logo.png" alt="Torch Logo" align="right">Torch - Using Laravel's Illuminate Components Independently
+
+[&lt;&lt; Back to Torch](../../readme.md)
+
+# Config
```

Comment:
You left the page title the same for each; you'll want this to say "Cache" on this one, etc.

Reply:
Fixed the page titles , sorry was distracted .